pipeline/coordinator.go

```diff
@@ -384,6 +393,14 @@ func (c *Coordinator) startOneUploadJob(p UploadJobPayload, handler Handler, for
 		catalystRegion:        os.Getenv("MY_REGION"),
 		sourceCodecVideo:      videoCodec,
 		sourceCodecAudio:      audioCodec,
+		sourceWidth:           videoTrack.Width,
```

Comment:
Should we just reuse the video.InputVideo struct here? I think most of these fields exist there as well.

Reply:
Yeah, I was thinking that as I was working on it - it's basically 1:1 now we're getting everything from ffprobe